README.md

```diff
@@ -17,6 +17,10 @@ Solutions to many common issues may be found at the [OSF Developer Docs](http://
 These instructions should work on Mac OSX >= 10.7
 
 - Create your virtualenv.
+```
+virtualenv osfenv
```

Comment:
Could we change the environment name to `env`, which is already listed in our `.gitignore`? That way, we prevent a dev from inadvertently checking in their virtualenv.